### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by tplavcic**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| upgrade-haproxy | passed |
| scheduled-backup | passed |
| scaling | passed |
| haproxy | passed |
| big-data | passed |
| storage | passed |
| self-healing | passed |
| self-healing-chaos | failed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | failed |
| demand-backup | passed |
| security-context | passed |
| smart-update | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade-consistency | passed |
| users | passed |
| pitr | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1071/commits/eb22b7f9c2756cf0bd6e53fe276cf36cfe318817
image: `perconalab/percona-xtradb-cluster-operator:PR-1071-eb22b7f9`

File: deploy/bundle.yaml [hunk 1]
```diff
@@ -759,14 +760,21 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: percona-xtradb-cluster-operator
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: percona/percona-xtradb-cluster-operator:1.10.0
```
Comment (hors) [inline deploy/bundle.yaml:763]: but we need to use main tag

Comment (tplavcic) [reply deploy/bundle.yaml:763]: Correct, but I will make another PR with changes needed after release. It was done for PSMDB the same: https://github.com/percona/percona-server-mongodb-operator/commit/9010c36fa1ca3f50069cb3240bc449e7119496a6

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| haproxy | passed |
| big-data | passed |
| storage | passed |
| scheduled-backup | failed |
| recreate | passed |
| scaling | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| security-context | failed |
| upgrade-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| limits | passed |
| smart-update | passed |
| demand-backup | failed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| monitoring-2-0 | failed |
| pitr | failed |
| affinity | passed |
| demand-backup-encrypted-with-tls | failed |
| one-pod | passed |
| auto-tuning | passed |
| upgrade-consistency | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1071/commits/35b68296e3c43fbae0aa2c6e555335a3c0f4e495
image: `perconalab/percona-xtradb-cluster-operator:PR-1071-35b68296`

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | failed |
| recreate | passed |
| storage | passed |
| restore-to-encrypted-cluster | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| operator-self-healing | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| operator-self-healing-chaos | passed |
| scaling-proxysql | passed |
| security-context | passed |
| haproxy | passed |
| big-data | passed |
| init-deploy | passed |
| limits | passed |
| monitoring-2-0 | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| demand-backup | passed |
| validation-hook | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1071/commits/35b68296e3c43fbae0aa2c6e555335a3c0f4e495
image: `perconalab/percona-xtradb-cluster-operator:PR-1071-35b68296`

**Pull request merged**